src/components/Bookstore.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import BookTitle from './Authors';
+import Input from './Form';
+import '../styles/App.css';
+
+function StoreBook() {
+  const Books = [
+    {
+      id: 1,
+      title: 'The Hunger Games',
+      genres: 'Action',
+      author: 'Suzanne Collins',
+      progres: 64,
+      chapter: 'Chapter 17: "A Lesson Learned"',
+    },
+    {
+      id: 2,
+      title: 'Dune',
+      genres: 'Fantasy',
+      author: 'Frank Herbert',
+      progres: 8,
+      chapter: 'Chapter 3: "A Lesson Learned"',
+    },
+    {
+      id: 3,
+      title: 'Capital in the Twenty-First Century',
+      genres: 'Economics',
+      author: 'Suzanne Collins',
+      progres: 0,
+      chapter: 'Introduction: "A Lesson Learned"',
+    },
+  ];
+
+  return (
+    <div>
+      <ul className="the-books">
+        {Books.map((book) => (
+          <li className="container" key={book.id}>
+            <BookTitle title={book.title} author={book.author} />
+            <button type="submit">Remove</button>
+          </li>
```

Comment:
- [ ] The remove button is supposed to be inside the single book component. In your case, that's the BookTitle component inside the Authors.js file. Kindly move this button into that component. 👍🏽

[ Optional Change ]
- [ ] It is s good convention to name your files and your components the same name. For instance, your BookTitle component could better be named Book, and the file name also is better named Book.js. This makes your code better organized and easy to read, find and solve bugs.  👍🏽